operator_ui/src/components/JobRuns/StatusIcon.js

```diff
@@ -3,9 +3,15 @@ import SuccessIcon from 'components/Icons/Success'
 import ErrorIcon from 'components/Icons/Error'
 import PendingIcon from 'components/Icons/Pending'
 
-const StatusIcon = ({ children, width, height }) => {
+const StatusIcon = ({ children, width, height, successIconColor }) => {
   if (children === 'completed') {
-    return <SuccessIcon width={width} height={height} />
+    return (
+      <SuccessIcon
+        width={width}
+        height={height}
+        color={successIconColor}
```

Comment:
Is there a case where we want a different color for the success icon? It feels like that should be encapsulated within the `<SuccessIcon />` component and should be pulled from the theme, rather than passed in as a prop.

Reply:
All that was an opinionated visual change of mine anyway, so I've decided to opt out from it. Ptal again @rupurt